peer/brontide.go

```diff
@@ -1159,6 +1210,12 @@ func (p *Brontide) Disconnect(reason error) {
 	p.cfg.Conn.Close()
 
 	close(p.quit)
+
+	if err := p.pingManager.Stop(); err != nil {
+		p.log.Errorf("couldn't stop pingManager during disconnect: %v",
+			err,
+		)
```

Comment:
nit: logs dont follow the same style rules as function calls. So can just put in 2 lines:
```
	p.log.Errorf("couldn't stop pingManager during disconnect: " +
			"%v", err)
```